src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,48 @@
-export const Tabs = () => {};
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (newId: string) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId);
+
+  if (!selectedTab) {
+    onTabSelected('tab-1');
+  }
+
+  const hendlerOnTabSelected = (tabId: string) => {
```

Comment:
Typo in the function name 'hendlerOnTabSelected', it should be 'handlerOnTabSelected'.